tests/test_metric.py

```diff
@@ -21,7 +21,7 @@ def _info(self):
             features=Features({"predictions": Value("int64"), "references": Value("int64")}),
         )
 
-    def _compute(self, predictions, references):
+    def _compute(self, predictions, references, **kwargs):
```

Comment:
why is this change needed?

Reply:
Really only so that I can do

```py
self.args = {"hello": 1, "world": 2}
self.result = self.metric.compute(**self.args)
```

in the `setUp` for the test cases. The args don't actually get used in the DummyMetric, but I wanted to make it obvious in the tests that the `metric_args` are the `args` that get passed to `compute()`. Looking at it now I think it's just confusing, and since I'm going to document it in the docstring anyways it's probably not useful. I'll revert it!